### ISSUE TITLE
twitch.tv ad stub

### ISSUE THREAD

**Issue by Xost3r** [AdguardTeam/AdGuardExtra#422]
sorry i am writing using google translate

1-While watching a broadcast on twitch, a wave of advertisements suddenly comes and adguard blocks the extra advertisement, but instead of the advertisement, a screen like the video appears, which is no different from the advertisement.

The second problem is that the broadcast freezes after a while, but there is no problem with the internet. You can see more details in the video.

vid: https://drive.google.com/file/d/1AJjm8RlJTAVghMiIYD3wvxcbubPm4rrK

**Comment by stanislav-atr** [comment]
@Xost3r, hi! This advertisement stub is known and yet to be reproduced. It seems that twitch only applies some of the ads to a small portion of users.

@AdamWr @zloyden check this out, please.

**Comment by AdamWr** [comment]
I cannot reproduce it at the moment, but I occasionally see this message.

**Comment by Zoxjib** [comment]
Having the same issue now. I've had it recur many times this week while a stream is running in the background for many hours (twitch drops). 

Switching to another stream seems to solve the problem.

**Comment by lejifps** [comment]
+++ got the same issue

also my streams are laggy puffering every few seconds ... driving me nuts

**Comment by Mo-norn** [comment]
Is this still difficult to reproduce?
It seems to be reproducible from a South Korea IP.
I was also able to reproduce it in browser incognito mode, not signed in.
I recently disconnected Adguard to see if it was fixed, but I was able to watch Twitch without any issues. I don't think there were any Twitch country ads during that time.
It may be difficult to reproduce perfectly in a short period of time because it depends on the duration of the ad or the streamer's ad settings.
I think it should be reproducible with a South Korea IP and a Korean streamers.
